ouroboros-consensus/src/Ouroboros/Consensus/Util/ThreadRegistry.hs

```diff
@@ -105,8 +105,10 @@ forkLinked :: forall m a. (MonadAsync m, MonadFork m, MonadMask m)
 forkLinked threadRegistry action = do
     me <- myThreadId
     fork threadRegistry $
-      action `catch` \(e :: SomeException) -> do
-        unless (isCancel (Proxy @m) e) $ throwTo me e
+      action `catch` \e -> do
```

Comment:
This should have been part of the "Use a fixed AsyncCancelled exception type" commit.

Reply:
It is. That's the only patch that touches this file.

Reply:
Oh, sorry, I was reviewing this commit per commit, but I think I accidentally opened a tab that showed all the changes, and I got confused for a second.

Reply:
no probs :-)